examples/consumer/main.go

```diff
@@ -206,41 +206,42 @@ func pingForStatus(resourceID string) {
 	}
 }
 
+func fastHTTPHandler(ctx *fasthttp.RequestCtx) {
+	fmt.Fprintf(ctx, "Hi there! RequestURI is %q", ctx.RequestURI())
```

Comment:
remove this is print

Reply:
no not until code freeze on friday..

Reply:
done

Reply:
@aneeshkp I think we can include this as part of Redfish Hw Event feature so it is targeted for 4.10 and does not need bug tracking for merging after code freeze.